### PR TITLE
Add bytes option for pkg-stats(8)

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Add bytes option for pkg-stats(8)

### CODE REVIEW

File: salt/modules/pkgng.py [hunk 1]
```diff
@@ -475,7 +475,7 @@ def update_package_site(new_url):
     return True
 
 
-def stats(local=False, remote=False, jail=None, chroot=None, root=None):
+def stats(local=False, remote=False, bytes=False, jail=None, chroot=None, root=None):
```
Comment (s0undt3ch) [inline salt/modules/pkgng.py:478]: Hmm, `bytes` is a python built-in function....
I wonder if this works of if we'll mess something up....

Comment (s0undt3ch) [reply salt/modules/pkgng.py:478]: `unit="bytes"`?

Comment (s0undt3ch) [reply salt/modules/pkgng.py:478]: What units are accepted?

Comment (krionbsd) [reply salt/modules/pkgng.py:478]: > What units are accepted?

bytes or human readable format according to IEC 80000-3 standards: `Ki/Mi/Gi/Ti/Pi/Ei`

Comment (s0undt3ch) [reply salt/modules/pkgng.py:478]: So you can pass all of those values?

Comment (krionbsd) [reply salt/modules/pkgng.py:478]: No, you might pass only bytes, by default it's human readable format

Comment (s0undt3ch) [reply salt/modules/pkgng.py:478]: Well, if the test suite didn't choke on it, I guess its fine being bytes

**Review by s0undt3ch — DISMISSED** [review]

File: salt/modules/pkgng.py (hunk 1 above)
Comment (Ch3LL) [inline salt/modules/pkgng.py:478]: Can we add this kwarg at the end in case someone is calling this function with positional args?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by Ch3LL — CHANGES_REQUESTED** [review]
also changelog entry required as well

**Comment by krionbsd** [comment]
> also changelog entry required as well

Sure, looks good now

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**